src/mono/wasm/debugger/DebuggerTestSuite/SetNextIpTests.cs

```diff
@@ -112,7 +112,11 @@ public async Task AsyncMethod()
             });
     }
 
+# if FEATURE_WASM_THREADS
```

Comment:
There's an internal property `System.Threading.Thread.IsThreadStartSupported` that is false on single-threaded wasm and true on multi-threaded and on all other platforms.  So we can peek at it using reflection and add a check to PlatformDetection

Reply:
Fixing the ActiveIssue is here: https://github.com/dotnet/arcade/pull/13634. I am thinking about merging this PR now and once the fix will be there, we can apply @lambdageek's idea and do it properly. For now, it would be nice to clean the CI.